docs/guides/high-availability/deploy-aws-accelerator-loadbalancer.adoc

```diff
@@ -0,0 +1,294 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Deploy an AWS Global Accelerator loadbalancer"
+summary="Building block for a loadbalancer"
+tileVisible="false" >
+
+This topic describes the procedure required to deploy an AWS global accelerator to route traffic between multi-site Keycloak deployments.
+
+include::partials/blueprint-disclaimer.adoc[]
+
+== Audience
+
+This guide describes how to deploy an AWS Global Accelerator instance to handle Keycloak client connection failover for multiple
+availability-zone Keycloak deployments.
+
+== Architecture
+
+All Keycloak client requests are routed via an AWS Global Accelerator instance. It's the responsibility of the configured
+accelerator to ensure that requests are routed to Keycloak clusters deployed in both availability-zones. Utilising an
+AWS Global Accelerator prevents clients from erroneously caching IP addresses of failed Keycloak deployments which is
+possible with DNS based failover strategies.
```

Comment:
We put more emphasis first on the concepts, and less on the specific implementation. 

The concepts it supports: Routing traffic between availability zones while keeping the same IP address for the clients on failover. The implementation that supports this on AWS happens to be the Global Accelerator.

Reply:
We actually didn't do this before with the Route53 docs, but I agree it's a better format :+1:

Reply:
I know it is a new requirement, I hope you don't mind. If you would rather handle this in a follow-up PR, this is also fine.